### ISSUE TITLE
Nginx issues and GD is missing.

### ISSUE THREAD

**Issue by jhughes2112** [woahbase/alpine-akaunting#2]
Hey guys, I really appreciate your efforts.  I've been trying to use this docker install for a few days and found a number of small issues that made it really, really hard to get going.

First, the nginx config file needs to have a much longer timeout.  60 seconds is not enough for it to create the database, so I bumped mine up to 300 seconds in both the http and server config blocks.

Second, once you actually get to the point of trying to make an invoice as a PDF or printout, it requires GD, which based on the current PHP 7.2 install should be the  php7-gd package. 

Dunno if there are other issues coming, but the install seems to work more or less once I got it going.  I have my own docker-compose, so my setup is a little different, but overall runs great.

Thanks again!

**Comment by jhughes2112** [comment]
Found another missing package.  It also needs php7-dom to produce PDF files.

**Comment by lapnapra** [comment]
Hi,

Thanks for letting us know. Gotta love issues like this that also have the solution, (and test if you say it's working as it should for you). So thanks for that too. :D  

I've added the packages but decided **not** to change the default Nginx configuration as I'd rather anyone using this image to for any real purpose actually update and manage their own configurations according to their use case.

Let me know if you have any more issues after pulling the new one.

**Comment by jhughes2112** [comment]
Rather than fork and put up my own, I'm happy to use yours.  Thanks again for maintaining it!

I pulled your update today and relaunched it, and both PDF and print functions work fine now.

The down-side about how Akaunting is set up is that if you have a database connected to it and get MOST of the way through the install wizard (step 2 will fail after nginx disconnects at 60 seconds), it leaves the DB in an unusable state AND thinks it completed step 2 and 3, which it hasn't, so it's a broken install.  I probably reinstalled it fifty times until I figured out nginx was cutting my connection.  A note about that, and to completely wipe and restart install BOTH of the DB and Akaunting (it writes an .env file too early in the process), would probably save other folks a lot of time.  This package is not very robust, I'm finding.

Anyway, current build is working great.  Thanks again.
